src/event.js

```diff
@@ -424,8 +426,11 @@ jQuery.event = {
 			delegateCount = handlers.delegateCount,
 			cur = event.target;
 
+		// Support (at least): Chrome, IE9
```

Comment:
The ticket reporting the original bug at http://bugs.jquery.com/ticket/13180 has a test case http://jsbin.com/welcome/71517/ that still blows up for IE11 but seems okay on Chrome and Firefox. For master at least, a `// Support: IE9+` would be correct?